tutorials-v5/lectures/Lecture-6-Quantum-Monte-Carlo-Trajectories.md

```diff
@@ -116,8 +116,11 @@ Here we start the Monte-Carlo simulation, and we request expectation values of p
 
 ```python
 ntraj = [1, 5, 15, 904]  # list of number of trajectories to avg. over
+mc = []
 
-mc = mcsolve(H, psi0, tlist, c_op_list, [a.dag() * a], ntraj)
+for n in ntraj:
+    result = mcsolve(H, psi0, tlist, c_op_list, e_ops=[a.dag() * a], ntraj=n)
+    mc.append(result)
```

Comment:
Then is removing the method `expect_traj_avg` an option? I don't see much other use than these examples.

Reply:
I'm happy with removing  `expect_trag_avg`. It feels like it would be better to have ways to slice or combine results and the cost of running `mcsolve` multiple times  can be made quite low if one uses the the `MCSolve` class.

Reply:
We added adding `mcsolve` result object to each other some time ago. I could work for this.